### ISSUE TITLE
Is it normal ? alert Unable to validate ............ State, format [uint8]

### ISSUE THREAD

**Issue by fennec622** [arachnetech/homebridge-mqttthing#426]
Hi 

I have a lot of error with integer numbers 

Is it normal ? 

`Apr  7 23:30:02 raspberrypiManager homebridge[1213]: [07/04/2021 à 23:30:02] [Alarm] Received NA - currentState state is now 2
Apr  7 23:30:02 raspberrypiManager homebridge[1213]: [07/04/2021 à 23:30:02] [Alarm] Unable to validate Security System Current State, format [uint8] - {"_events":{},"_eventsCount":3,"iid":10,"value":3,"status":null,"statusCode":0,"subscriptions":5,"displayName":"Security System Current State","UUID":"00000066-0000-1000-8000-0026BB765291","props":{"format":"uint8","perms":["ev","pr"],"minValue":0,"maxValue":4,"minStep":1,"validValues":[0,1,2,3,4]}}

Apr  7 23:30:02 raspberrypiManager homebridge[1213]: [07/04/2021 à 23:30:02] [Alarm] Received NA - currentState state is now 2
Apr  7 23:30:02 raspberrypiManager homebridge[1213]: [07/04/2021 à 23:30:02] [Alarm] Unable to validate Security System Current State, format [uint8] - {"_events":{},"_eventsCount":3,"iid":10,"value":3,"status":null,"statusCode":0,"subscriptions":5,"displayName":"Security System Current State","UUID":"00000066-0000-1000-8000-0026BB765291","props":{"format":"uint8","perms":["ev","pr"],"minValue":0,"maxValue":4,"minStep":1,"validValues":[0,1,2,3,4]}}

Apr  7 21:15:14 raspberrypiManager homebridge[1213]: [07/04/2021 à 21:15:14] [Volet Cuisine] Received STOPPED - positionState state is now 2
Apr  7 21:15:14 raspberrypiManager homebridge[1213]: [07/04/2021 à 21:15:14] [Volet Cuisine] Unable to validate Position State, format [uint8] - {"_events":{},"_eventsCount":3,"iid":12,"value":0,"status":null,"statusCode":0,"subscriptions":1,"displayName":"Position State","UUID":"00000072-0000-1000-8000-0026BB765291","props":{"format":"uint8","perms":["ev","pr"],"minValue":0,"maxValue":2,"minStep":1,"validValues":[0,1,2]}}`

Before update mqtthing no error 

i have no change configuration 

thanks

**Comment by plantoschka** [comment]
I get similiar warnings since the last update.

Everything seems to work fine.

`[4/8/2021, 2:28:55 PM] [Fenster Links Kind] Unable to validate Contact Sensor State, format [uint8] - {"_events":{"change":[null,null]},"_eventsCount":3,"iid":10,"value":1,"status":null,"statusCode":0,"subscriptions":8,"displayName":"Contact Sensor State","UUID":"0000006A-0000-1000-8000-0026BB765291","props":{"format":"uint8","perms":["ev","pr"],"minValue":0,"maxValue":1,"minStep":1,"validValues":[0,1]}}
[4/8/2021, 2:29:05 PM] [Bewegungsmelder Bad] Unable to validate Occupancy Detected, format [uint8] - {"_events":{},"_eventsCount":3,"iid":10,"value":0,"status":null,"statusCode":0,"subscriptions":8,"displayName":"Occupancy Detected","UUID":"00000071-0000-1000-8000-0026BB765291","props":{"format":"uint8","perms":["ev","pr"],"minValue":0,"maxValue":1,"minStep":1,"validValues":[0,1]}}
[4/8/2021, 2:29:10 PM] [Bewegungsmelder Bad] Unable to validate Occupancy Detected, format [uint8] - {"_events":{},"_eventsCount":3,"iid":10,"value":1,"status":null,"statusCode":0,"subscriptions":8,"displayName":"Occupancy Detected","UUID":"00000071-0000-1000-8000-0026BB765291","props":{"format":"uint8","perms":["ev","pr"],"minValue":0,"maxValue":1,"minStep":1,"validValues":[0,1]}}`

**Comment by fennec622** [comment]
Yes all is ok
Just don't like error 😂

**Comment by JuniorGenius** [comment]
I also have this issue with various devices, but it also seems as though everything is working properly.

```
[Occupancy Sensor] Unable to validate Occupancy Detected, format [uint8] - {"_events":{},"_eventsCount":3,"iid":10,"value":1,"status":null,"statusCode":0,"subscriptions":2,"displayName":"Occupancy Detected","UUID":"00000071-0000-1000-8000-0026BB765291","props":{"format":"uint8","perms":["ev","pr"],"minValue":0,"maxValue":1,"minStep":1,"validValues":[0,1]}}
[H1 Switch] Unable to validate Battery Level, format [uint8] - {"_events":{},"_eventsCount":3,"iid":54,"value":0,"status":null,"statusCode":0,"subscriptions":2,"displayName":"Battery Level","UUID":"00000068-0000-1000-8000-0026BB765291","props":{"format":"uint8","perms":["ev","pr"],"unit":"percentage","minValue":0,"maxValue":100,"minStep":1}}
```

**Comment by bubez81** [comment]
maybe i found the cause of the problem...it could be homeassistant and homekit integration

**Comment by kerberjg** [comment]
I also get it, here's the config of my device(s)

```
  {
            "accessory": "mqttthing",
            "type": "windowCovering",
            "name": "SC500W-4368",
            "url": "mqtt://localhost:1883",
            "username": "***",
            "password": "***",
            "topics": {
                "getCurrentPosition": {
                    "topic": "stat/tasmota_F03110/RESULT",
                    "apply": "return JSON.parse(message).Shutter1.Position;"
                },
                "setTargetPosition": "cmnd/tasmota_F03110/shutterposition1",
                "getTargetPosition": {
                    "topic": "stat/tasmota_F03110/RESULT",
                    "apply": "return JSON.parse(message).Shutter1.Position;"
                },
                "getPositionState": {
                    "topic": "stat/tasmota_F03110/RESULT",
                    "apply": "return JSON.parse(message).Shutter1.Direction;"
                }
            },
            "positionStateValues": [
                "-1",
                "1",
                "0"
            ],
            "startPub": {
                "cmnd/tasmota_F03110/shutterposition1": ""
            }
        },
        {
            "accessory": "mqttthing",
            "type": "windowCovering",
            "name": "SC500W-1992",
            "url": "mqtt://localhost:1883",
            "username": "***",
            "password": "***",
            "topics": {
                "getCurrentPosition": {
                    "topic": "stat/tasmota_EFA798/RESULT",
                    "apply": "return JSON.parse(message).Shutter1.Position;"
                },
                "setTargetPosition": "cmnd/tasmota_EFA798/shutterposition1",
                "getTargetPosition": {
                    "topic": "stat/tasmota_EFA798/RESULT",
                    "apply": "return JSON.parse(message).Shutter1.Position;"
                },
                "getPositionState": {
                    "topic": "stat/tasmota_EFA798/RESULT",
                    "apply": "return JSON.parse(message).Shutter1.Direction;"
                }
            },
            "positionStateValues": [
                "-1",
                "1",
                "0"
            ],
            "startPub": {
                "cmnd/tasmota_EFA798/shutterposition1": ""
            }
        }
```

**Comment by arachnetech** [comment]
Really sorry for my delay in responding to these issues - work has been crazily busy. Hoping to make some changes to help this weekend...

**Comment by arachnetech** [comment]
Should be fixed in 1.1.28

**Comment by kerberjg** [comment]
No worries, thanks for your great work!

I'll try out the update in a few days and let you know how it goes :)
